source/install/tarball.rst

```diff
@@ -11,17 +11,17 @@ You can find links to the binary tarballs from the `Percona website`_
 
    .. code-block:: bash
 
-      $ wget https://www.percona.com/downloads/percona-server-mongodb-4.0/\
-      percona-server-mongodb-4.0.4-1/binary/debian/jessie/x86_64/\
-      percona-server-mongodb-4.0.4-1-ra1cb178-jessie-x86_64-bundle.tar
-      $ tar xfz percona-server-mongodb-4.0.4-1-ra1cb178-jessie-x86_64-bundle.tar
+      $ wget https://www.percona.com/downloads/percona-server-mongodb-4.2/\
+      percona-server-mongodb-4.2.0-1/binary/debian/jessie/x86_64/\
```

Comment:
debian jessie binary not available, maybe put one from ubuntu bionic instead: percona-server-mongodb-4.2.0-1-bionic-x86_64.tar.gz
https://jenkins.percona.com/job/percona-server-mongodb-4.2-TARBALL-release/label_exp=min-bionic-x64/

Reply:
First two lines still have Debian Jessie binary tarball link if I'm not missing something.